Dockerfile-sc

```diff
@@ -0,0 +1,45 @@
+FROM ruby:2.4.2
```

Comment:
This is the docker file to build the docker image that will do the development. It starts with the base ruby image for the ruby version currently used in the project,  but for easy forward compatibility it installs rbenv as well.